### DISCUSSION TITLE
Night mode for documentation

### DISCUSSION THREAD

**Discussion by benixmaxedis** [excaliburjs/Excalibur#2657] [Ideas]
Would be nice if there was night mode for documentation

**Comment by eonarheim** [comment]
@benixmaxedis Totally agree!

@kamranayub @jedeen How hard do you think this would be in the current setup? Should we look at the site rebuild first?

**Reply by kamranayub** [reply]
I don't think it would be too tough, though not sure with Semantic UI. If we switch to Tailwind, which I'd prefer, then it would be simple 🙌 If we decide to use a different docs platform or something that would be built in, ala Docusaurus.
